src/components/v2/shared/SplitItem.tsx

```diff
@@ -32,137 +30,139 @@ const LockedText = ({ lockedUntil }: { lockedUntil: number }) => {
   )
 }
 
-export default function SplitItem({
-  split,
-  showSplitValue,
-  currency,
-  totalValue,
+const JuiceboxProjectBeneficiary = ({
   projectOwnerAddress,
-  valueSuffix,
-  valueFormatProps,
-  reservedRate,
+  split,
 }: {
   split: Split
-  currency?: BigNumber
-  totalValue: BigNumber | undefined
   projectOwnerAddress: string | undefined
-  showSplitValue: boolean
-  valueSuffix?: string | JSX.Element
-  valueFormatProps?: { precision?: number }
-  reservedRate?: number
-}) {
+}) => {
   const {
     theme: { colors },
   } = useContext(ThemeContext)
-  const isMobile = useMobile()
 
-  const { data: handle } = split.projectId
-    ? useProjectHandle({ projectId: parseInt(split.projectId) })
-    : { data: undefined }
+  if (!split.projectId) return null
 
   const isProjectOwner = projectOwnerAddress === split.beneficiary
-  const isJuiceboxProject = split.projectId
-    ? BigNumber.from(split.projectId).gt(0)
-    : false
-  const itemFontSize = isMobile ? '0.9rem' : 'unset'
-
-  const JuiceboxProjectBeneficiary = () => {
```

Comment:
Having these as closures was causing hella issues. Moved them out of the `SplitItem` scope and gave them props.